invenio_theme/ext.py

```diff
@@ -81,9 +71,9 @@ def init_app(self, app, **kwargs):
         # Register context processor
         @app.context_processor
         def _theme_icon_ctx_processor():
-            from invenio_theme.proxies import current_theme_icons
+            return {"current_theme_icons": current_theme_icons}
```

Comment:
i thought i remove the `current_` prefix, because the menu is not longer dependent on the context. (yes it was for the icons but it is not true there either, because we removed that part)